src/utils/reeds_data_parsing.jl

```diff
@@ -480,8 +774,12 @@ end
         a Gen_Storage struct containing information about generators/storage
         technologies for each region.
 """
-function process_genstors(regions::Vector{<:AbstractString}, timesteps::Int)
-    gen_stors = Gen_Storage[] # empty for now
+function process_genstors(gen_stors, regions::Vector{<:AbstractString}, timesteps::Int)
+    if length(gen_stors)
```

Comment:
This is checking if `gen_stors` has been populated earlier, and if so just returning it. If not, since we do not make `gen_stors` for any generators other than HD reservoirs, we just return empty.

Reply:
Okay. Do we need this functionm anymore after all the updates now?

Reply:
I think we should keep it as a place holder to be sure we implement it if and when such data/devices become available.